iridauploader/core/upload.py

```diff
@@ -0,0 +1,253 @@
+"""
```

Comment:
github is not showing it, but this file was cli_entry.py, renamed and heavily edited.